### DISCUSSION TITLE
Search results often don't display the source page as a title?

### DISCUSSION THREAD

**Discussion by YozoraXCII** [squidfunk/mkdocs-material#7580] [Q&A] [answered]
I'm not sure if this is a "me" issue, so I chose to open it as a Discussion rather than an Issue.

When I use search, I often find that the results do not show what page the result is from - for example if I search for "test" on my website I get this:

![image](https://github.com/user-attachments/assets/b790aaf1-a77c-4aa7-9af9-93723324fabb)

Ideally all of these results would show me the page that the extract has come from.

I don't know if this is because I'm doing something incorrect with headers, or if I have a rogue setting or CSS or something.

Steps to reproduce:

1. Visit https://kometa.wiki/
2. Search for "test"

**Comment by kamilkrzyskow** [comment, marked as answer]
Hi @YozoraXCII,
there is some rogue CSS on your index page. Only the home page has this issue.

Notice how your other pages work fine:
- https://kometa.wiki/en/latest/config/overview/

![image](https://github.com/user-attachments/assets/630ec015-4d9e-4926-99ae-b3bcd8a9ed4a)


https://github.com/Kometa-Team/Kometa/blob/83dd84c2f2627a5f0f21af4009a1bc0e93ad420a/docs/index.md?plain=1#L3-L8
The issue is the selector with the `.md-typeset h1` that detects every `h1` on the page.
Scope it to `.md-content` with the selector.

**Reply by YozoraXCII** [reply]
Thank you for this!

I think this was added to stop the initial header from appearing on the homepage, as it was just showing up as a `#`, I'll see if I can find another way to resolve that.

**Reply by kamilkrzyskow** [reply]
Like I mentioned scope it to `.md-content`:
```css
  .md-content h1,
  .md-content__button {
    display: none;
  }
```
